### DISCUSSION TITLE
What's the equivalent of InvokeUnmarshalled from JS to .Net?

### DISCUSSION THREAD

**Discussion by user72356** [dotnet/aspnetcore#26262] [General] [answered]
Blazor JSInterop has a special JSRuntime that lets me call JS functions from .Net without any marshalling for max performance in a webassembly scenario: [WebAssemblyJSRuntime Class](https://docs.microsoft.com/en-us/dotnet/api/microsoft.jsinterop.webassembly.webassemblyjsruntime?view=aspnetcore-3.1)

What is the equivalent in JS for calling a .Net method from JS without marshalling?

**Comment by SteveSandersonMS** [comment, marked as answer]
Great question!

I've posted an answer at https://github.com/dotnet/aspnetcore/issues/26287#issuecomment-700150823.
